meetings/2024-04-24.md

```diff
@@ -28,41 +35,101 @@
+* VB: We can ask on the mailing list and take it from there.
+* SC: While there are various individuals working on both, I think it's on them to make the case. We have limited bandwidth to dedicate to features. This is the incubation space, we shouldn't wait for the working group and try to bring it directly there out of thin air. We can make a call, inviting people to make a PR, whether like they want to change WebID to DID, or propose new method, or whatever.
+* eP: Would like to emphasize designing solid in a way tht is future-proof. wWe can do our best to make it easier to introduce in the future. We want to avoid breaking changes in the future.
+* SC: If something is preventing the use of DIDs in Solid, the proponents of DID should make that case. I don't see why we need to do that work.
+* eP: We need to be careful how we evaluate people are not bringing things into the community. Some people might think it's painful/not productive and prefer to work outside. Just because they don't bring to CG doesn't mean they don't care.
+* SC: That's architecutre astronomy. We can't abstract everything just in case.
+* HZ: The issue was brought to the CG, there are two issue, second marked as a duplicate. We should close those issues one way or another. No one rejects the idea, maybe next step would be STM making sure inviting the people with interest and expertise to that meeting.
+
+ACTION: HZ to post to mailing lists, Solid and DID, inviting people to comment on the topic of DID. 
+
+### WG Chairs
+
+* SC: I understand that Eric brings a lot of experience. Last I know he is affiliated with Janeiro Digital. There is one issue, some people with the same affiliation, as well as of other proposed chair with another affiliation were trying to shortcut process of ammending the charter without Solid CG's consensus: https://github.com/solid/solid-wg-charter/pull/47 . I trust Eric and I know he knows W3C rules. I wonder if the two original proposed chairs are being changed. Many objections were about those chairs. If the already proposed 2 chairs lack experience, what do they bring to the table?
+* PAC: I don't think they bring anything that would not be offered by other possible chairs. The objections raised by a number of AC reviewers, part of them were about W3C experience, adding Eric addresses that. Eric's association with Janeiro ended before the events you mention. I'm not sure but it might be not fair to have it weight on him. I believe that the two proposed chairs bring different perspectives to the table. On is working in the private company that is implementing Solid servers, the other one working in the public sector. Eric has his own perspective working on the shapes for example. I don't think concerns that were raised were sufficient to change the agreement with two proposed chairs. I think that adding Eric addresses the most justified objection. 
```

Comment:
```suggestion
* SC: I understand that Eric brings a lot of experience. Last I knew, he is affiliated with Janeiro Digital. There is one issue, some people with the same affiliation, as well as of another proposed chair with another affiliation, were trying to shortcut process of amending the charter without Solid CG's consensus: <https://github.com/solid/solid-wg-charter/pull/47>. I trust Eric, and I know he knows W3C rules. I wonder if the two original proposed chairs are being changed? Many objections were about those chairs. If the already proposed 2 chairs lack experience, what _do_ they bring to the table?
* PAC: I don't think they bring anything that could not be offered by other possible chairs. The objections raised by a number of AC reviewers, part of them were about W3C experience; adding Eric addresses that. Eric's association with Janeiro ended before the events you mention. I'm not sure but it might be not fair to have it weigh on him. I believe that the two proposed chairs bring different perspectives to the table. One is working in the private company that is implementing Solid servers, the other one working in the public sector. Eric has his own perspective working on the shapes, for example. I don't think concerns that were raised were sufficient to change the agreement with two proposed chairs. I think that adding Eric addresses the most justified objection. 
```